where-to-travel/src/main/webapp/index.html

```diff
@@ -122,6 +123,36 @@ <h5> Finding Attractions... </h5>
           </div>
         </div>
       </div>
+        
+      <div class="modal fade" id="signUp-modal" tabindex="-1" role="dialog" aria-hidden="true">
+        <div class="modal-dialog modal-sm" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h5 class="modal-title">Sign Up</h5>      
+            </div>
+            <div class="modal-body">
+              <form style="margin: auto" id="signUpForm" action="">
+                <div class="form-group">
+                    <label for="emailInput" class="bmd-label-floating">Email address</label>
+                    <input type="email" class="form-control" id="emailSignUp">
+                </div>
+                <div class="form-group">
+                    <label for="passwordSignUp" class="bmd-label-floating">Password</label>
+                    <input type="password" class="form-control" minlength="6" id="passwordSignUp" oninvalid="this.setCustomValidity('Password must be at least 6 characters.')">
+                </div>
+                <div class="form-group">
+                    <label for="passwordConfirmation" class="bmd-label-floating">Confirm password</label>
+                    <input type="password" class="form-control" id="passwordConfirmation">
+                </div>
+                <div class="text-center">
+                    <button type="submit" id="submit" class="btn btn-primary">Sign Up</button>
+                    <button type="button" id="back"  class="btn btn-primary" data-dismiss="modal">Cancel</button>
+                </div>
```

Comment:
Is this 2 spaces or 4? It looks like 4 but I could be wrong.

Reply:
You're right. I have fixed this in the latest commits.